rust/core-lib/src/movement.rs

```diff
@@ -12,12 +12,12 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-//! Representation and calculation of movement within a view.
+//! Representation and calculation of movement within a lo.
```

Comment:
Nitty, but IMHO it'd make sense to just say lineoffset here

Reply:
Fixed.